hidimstat/Dnn_learner_single.py

```diff
@@ -19,59 +19,62 @@
 )
 
 
-class DNN_learner_single(BaseEstimator):
+class Dnn_learner_single(BaseEstimator):
```

Comment:
```suggestion
class DnnLearnerSingle(BaseEstimator):
``` Class names have to be CamelCase